pontoon/translations/views.py

```diff
@@ -169,6 +173,14 @@ def create_translation(request):
                 description=desc,
             )
 
+    # Update Translation Champion Badge stats
+    if after_count < before_count:
+        badge_thresholds = [5, 50, 250, 1000]
+        for level, threshold in enumerate(badge_thresholds, start=1):
+            if after_count >= threshold and before_count < threshold:
+                # TODO: Send a notification to the user
+                print(f"New level achieved: Level {level}")
+
```

Comment:
Why do we need to get the `after_count` from the DB, when we know it's `before_count + 1`? Additionally, comparison in line 176 should be inverted, otherwise condition in line 180 will never be `True`.

I think we can simplify this to the following:
```suggestion
    # Award Translation Champion Badge stats
    if user.badges_translations_count in [5, 50, 250, 1000]:
        # TODO: Send a notification to the user
        pass
```

Tresholds should be defined in one place only, possibly in settings.